thirdparty/github.go

```diff
@@ -1616,7 +1606,6 @@ func getPRMergeBase(ctx context.Context, token string, data GithubPatch) (string
 		if err != nil {
 			return "", errors.Wrap(err, "getting installation token")
 		}
-		defer func() { _ = RevokeInstallationToken(ctx, token) }()
```

Comment:
Do we have some sort of token limit such that if a bunch of things are long running spawn new tokens and they aren't revoked after nothing is using it we're going to have a new issue?